test-cases/performance/perf-regression-latency-650gb-grow-shrink.yaml

```diff
@@ -35,3 +35,4 @@ send_email: true
 email_recipients: ["scylla-perf-results@scylladb.com"]
 use_prepared_loaders: true
 use_hdr_cs_histogram: true
+email_subject_postfix: 'latency during grow-shrink'
```

Comment:
i can be "calculated" automatically, based on existing content, not need to add such parameter in this test